src/app/components/shared/sensor-widget.service.test.js

```diff
@@ -21,16 +20,6 @@ describe('SensorWidgetService', () => {
         spyOn(sut.socket, 'emit').and.callThrough();
     });
 
-    describe('setup socket io', () => {
-        beforeEach(() => {
-            sut.setupSocket();
-        });
-
-        it('should setup socket io', () => {
-            expect(sut.io).toHaveBeenCalledWith(BACK_END_WS);
```

Comment:
I guess this test is important, because here we check setup socket io with proper url. 

What do you think about this or why you deleted this?